src/patternfly/components/Table/table.scss

```diff
@@ -115,7 +115,7 @@
   --#{$table}__action--PaddingRight: var(--pf-t--global--spacer--sm);
 
   // * Table expandable row
-  --#{$table}__expandable-row--Transition: var(--#{$pf-global}--Transition); // TODO: update to token when available
+  --#{$table}__expandable-row--Transition: var(--pf-t--global--transition); // TODO: update to token when available
```

Comment:
TODONE

Reply:
That would be a great productivity app name!